languages/python/exercises/concept/dicts/.docs/introduction.md

```diff
@@ -0,0 +1,3 @@
+A `dict` object in Python is a mutable\* mapping of unique keys to values. A key is basically a name for a value. A value can be any type of object, like `strings`, `ints`, `lists`, etc. Dictionaries have different methods to change them, like getting a value from a specific key in the dictionary, setting a default value for a key and many more. Dictionaries are also iterable by their keys.
```

Comment:
I'd link straight to the docs here.  [dicts](https://docs.python.org/3/library/stdtypes.html#typesmapping) and [mapping types](https://docs.python.org/3/library/stdtypes.html#typesmapping).  And since the docs bring up hashing, we probably should here too.  Something like _"A [`dict`](https://docs.python.org/3/library/stdtypes.html#typesmapping) object in Python is a mutable\* mapping of unique, hashable **keys** to values. ..."_